composer.json

```diff
@@ -6,12 +6,12 @@
   "require": {
     "php": ">=7.4",
     "ext-curl": "*",
-    "ext-json": "*"
+    "ext-json": "*",
+    "jtl/php-generic-collection": "^0.3.0",
+    "guzzlehttp/guzzle": "^6.5"
```

Comment:
There was a reason why we put generic-collection and guzzle to require-dev. This package is just a library and will never ever run as an standalone application. It will always a part (dependency) of any application.

So we want to make dependencies more visible by putting them to require-dev only - and also add them to the `suggest` section. This will force an Application to require the necessary requirement only in application scope.

Reply:
When I installed by composer, nothing does works. Because this package was missing jtl/php-generic-collection and guzzle I got already in vendors. Composer doesn't realize to load this package. I thought this was I mistake, that's why I move it up, because it's runtime libs and not development libs.

Reply:
ok, let's keep the dependencies in required section